rpcd/playbooks/verify-maas.yml

```diff
@@ -31,44 +31,26 @@
       file:
         path: "{{ maas_rpc_scripts_dir }}"
         state: directory
-    - name: copy script to all hosts
+    - name: Copy rpc-maas-tool to all hosts
       copy:
-        src: "{{ maas_rpc_scripts_dir }}/rpc-maas-tool.py"
-        dest: "{{ maas_rpc_scripts_dir }}/rpc-maas-tool.py"
+        src: "{{ maas_rpc_scripts_dir }}/{{item}}"
+        dest: "{{ maas_rpc_scripts_dir }}/{{item}}"
         mode: 0755
-    - name: "Verify Checks & Alarms are registered"
+      with_items:
+        - rpc-maas-tool.py
+        - alarmparser.py
+    - name: "Verify MaaS alarm criteria and plugins"
       shell: |
         {% if maas_venv_enabled | bool %}
         . {{ maas_venv_bin }}/activate
         {% endif %}
-        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
-        {% for ec in maas_excluded_checks %} --excludedcheck {{ec}}
-        {% endfor %}
+        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-local
       register: verify_maas
       failed_when: verify_maas.rc != 0
       changed_when: False
       until: verify_maas.rc == 0
       retries: "{{ verify_maas_retries }}"
       delay: "{{ verify_maas_delay }}"
 
-    - name: "Allow MAAS time to run checks before verifying their status"
```

Comment:
@sigmavirus24 Tags add complication because we then need to change the way that the playbooks are executed to include or exclude a tag.